srtcore/buffer_rcv.cpp

```diff
@@ -258,7 +258,7 @@ int CRcvBuffer::dropMessage(int32_t seqnolo, int32_t seqnohi, int32_t msgno, Dro
 {
     IF_RCVBUF_DEBUG(ScopedLog scoped_log);
     IF_RCVBUF_DEBUG(scoped_log.ss << "CRcvBuffer::dropMessage: seqnolo " << seqnolo << " seqnohi " << seqnohi
```

Comment:
If you already fix comments, you might fix to make it also shorter using the marker convention. For example:
```suggestion
    IF_RCVBUF_DEBUG(scoped_log.ss << "CRcvBuffer::dropMessage: %(" << seqnolo << "-" << seqnohi << ") "
```
... and